static/js/index.js

```diff
@@ -1,3 +1,13 @@
+function github(){
+  let updated = "";
+  for (let dictNum = 0; dictNum < 3; dictNum++){
```

Comment:
No need to assume you only get 3 commits here, the server should make that decision and you can just print out however many commits it sends you. So I'd suggest doing `dictNum < latestCommits.length` instead.

However, the preferred way to do this in JS would be to use forEach to do something like this:
```
latestCommits.forEach(commit => {
    updated += <blah blah>
});
```

Up to you which you prefer here.